### PR TITLE
Feature/9804 recycle bin/9872 screen

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Implements the new recycle bin screen. 
Also fixes a padding issue on the footer view controller that occurred when the button was shown after the view appeared.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9872

## Screenshots
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 13
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 20
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 27
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 32
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 42
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 48
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 43 58
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 44 03
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 44 21
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 44 25
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 44 41
Simulator Screen Shot - iPhone 13 Pro - 2021-10-13 at 18 44 59
Simulator Screen Shot - iPhone 13 Pro - 2021-10-14 at 17 53 15
Simulator Screen Shot - iPhone 13 Pro - 2021-10-14 at 17 53 27

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -3290,3 +3290,84 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
...
+
+"RecycleBin_CoronaTest_registrationDate" = "registriert am %@";
+
+"RecycleBin_CoronaTest_sampleCollectionDate" = "durchgeführt am %@";
+
+/* - Empty State */
+
+"RecycleBin_EmptyState_title" = "Ihr Papierkorb ist leer";
+
+"RecycleBin_EmptyState_description" = "Wenn Sie Zertifikate entfernen, werden sie in den Papierkorb verschoben und hier angezeigt.";
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:3341]: Aren't Tests also moved to the "Papierkorb" if they are deleted? Should this be mentioned there?

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/RecycleBin/RecycleBinViewController.swift [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+		tableView.separatorStyle = .none
+		tableView.rowHeight = UITableView.automaticDimension
+		tableView.estimatedRowHeight = 60
+		tableView.accessibilityIdentifier = AccessibilityIdentifiers.TraceLocation.Overview.tableView
+	}
+
+	private func animateCellHeightChanges() {
+		DispatchQueue.main.async { [self] in
+			tableView.performBatchUpdates(nil, completion: nil)
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/RecycleBin/RecycleBinViewController.swift:164]: Why is this needed?

**Review by ArturFriesen — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/RecycleBin/RecycleBinViewController.swift (hunk 2 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/RecycleBin/RecycleBinViewController.swift:164]: Without this when entering the editing mode the cells retain their height even though their width is reduced by the delete icon that appears on the left side and in some cases that meant that labels were cut off. With this code the height is adjusted when entering and leaving editing mode.
